packages/scoutgame-ui/src/components/claim/components/PointsClaimContainer.tsx

```diff
@@ -31,10 +32,18 @@ export async function PointsClaimContainer() {
     return null;
   }
 
-  const { bonusPartners, points, builders, repos } = data;
+  const { bonusPartners, points, builders, repos, partnerRewardPayoutCount } = data;
 
   const claimData = isOnchainApp ? (data as UnclaimedTokensSource).claimData : undefined;
 
+  const [unclaimedPartnerRewardsErr, unclaimedPartnerRewards] = await safeAwaitSSRData(
+    getUnclaimedPartnerRewards({ userId: scoutId })
+  );
+
+  if (unclaimedPartnerRewardsErr) {
+    return null;
```

Comment:
i dont think we should be returning nothing here?

Reply:
`safeAwaitSSRData` always logs the error inside its onError handler.